python/lsst/ctrl/mpexec/mock_task.py

```diff
@@ -68,8 +68,10 @@ def mockDatasetTypeName(cls, datasetTypeName: str) -> str:
         """Make mock dataset type name from actual dataset type name."""
         return "_mock_" + datasetTypeName
 
-    def _get(self, ref: Union[DeferredDatasetRef, DatasetRef]) -> Any:
+    def _get(self, ref: Optional[Union[DeferredDatasetRef, DatasetRef]]) -> Any:
```

Comment:
Yes, I had that fix and the other fix on this ticket on #199

Reply:
Ah OK, then I take this package out of the ticket.